shell/platform/android/io/flutter/app/FlutterApplication.java

```diff
@@ -16,7 +17,7 @@
  * <p>Using this {@link android.app.Application} is not required when using APIs in the package
  * {@code io.flutter.embedding.android} since they self-initialize on first use.
  */
-public class FlutterApplication extends Application {
+public class FlutterApplication extends SplitCompatApplication {
```

Comment:
We already started asking people to get rid of the FlutterApplication. If this is needed, we might have to add some docs for people to manually add it back or modify their existing Application superclass.

Reply:
There are other ways to accomplish this. This was just the simplest, but I can easily move to one of the other two methods that dont reply on this extension. See https://developer.android.com/guide/playcore/dynamic-delivery#enable_splitcompat